incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/utils/ObjectIntFunction.java

```diff
@@ -12,10 +12,10 @@
  * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package io.aklivity.zilla.runtime.exporter.prometheus.internal.utils;
+package io.aklivity.zilla.runtime.exporter.otlp.internal.utils;
```

Comment:
This should move to `io.aklivity.zilla.runtime.engine.util.function` in `engine` for potential reuse alongside other functional interfaces, unit test too.